blog/_posts/2018-07-07-Team101DaysOfSummerInroduction.md

```diff
@@ -0,0 +1,78 @@
+---
+title: The Beginning
+layout: post
+created_at: Sat 7 Jul 2018
+permalink: blog/2018-07-07-team-101-days-of-summer-intro
```

Comment:
Hi @paavininanda ! Thanks for the PR. Please see the comments below. 

Permalink and filename should be the same. You can check the old blog posts for an example.